MachObfuscator/DependencyAnalysis/ObfuscationPaths+Building.swift

```diff
@@ -1,20 +1,21 @@
 import Foundation
 
 extension ObfuscationPaths {
-    static func forAllExecutablesWithDependencies(inDirectory dir: URL, fileRepository: FileRepository = FileManager.default, dependencyNodeLoader: DependencyNodeLoader, obfuscableFilesFilter: ObfuscableFilesFilter) -> ObfuscationPaths {
+    static func forAllExecutablesWithDependencies(inDirectory dir: URL, fileRepository: FileRepository = FileManager.default, dependencyNodeLoader: DependencyNodeLoader, obfuscableFilesFilter: ObfuscableFilesFilter, withDependencies: Bool = true) -> ObfuscationPaths {
```

Comment:
What do you think about changing `withDependencies` to e.g. `deepSearch`? `withDependencies` is already a part of the method name, so setting this to false seems to be in contradiction with the method itself ;)

Reply:
so maybe lets remove `WithDependencies` from method name?

Reply:
Great idea! I love it.

Reply:
done